packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -179,14 +193,23 @@ export const ADD_AUCTION_MANIFEST = harden({
         auctioneer: true,
```

Comment:
don't need to consume the auctioneer installation

Reply:
done